addons/languages/coptic/pack/src/main/res/xml/coptic_swift.xml

```diff
@@ -0,0 +1,37 @@
+<Keyboard xmlns:android="http://schemas.android.com/apk/res/android" xmlns:ask="http://schemas.android.com/apk/res-auto" android:keyWidth="10.0%p" android:keyHeight="@integer/key_normal_height" ask:autoCap="false">
+  <Row>
+    <Key android:keyEdgeFlags="left" android:codes="ⲑ"/>
+    <Key android:codes="ϣ"/>
```

Comment:
Wow! I had no idea you could give literals for `android:codes`. That's *so* much easier than decimal codepoints!

Reply:
For sure, but it does get confusing with combining characters.